.gitignore

```diff
@@ -0,0 +1,5 @@
+# Generated by Maven
```

Comment:
This shouldn't be in the project - please remove

Reply:
Actually that's a manually written comment meaning "the directory below is generated by Maven".